src/main/java/world/bentobox/bentobox/managers/AddonsManager.java

```diff
@@ -130,7 +130,8 @@ private void loadAddon(@NonNull File f) {
             addon.setState(Addon.State.LOADED);
         } catch (NoClassDefFoundError | NoSuchMethodError | NoSuchFieldError e) {
             // Looks like the addon is incompatible, because it tries to refer to missing classes...
-            handleAddonIncompatibility(addon);
+            //handleAddonIncompatibility(addon);
+            e.printStackTrace();
```

Comment:
You could just add a `plugin#logStacktrace()` in handleAddonIncompatibility. Printing the stacktrace directly is to be avoided (https://rules.sonarsource.com/java/RSPEC-1148), and the handleAddonIncompatibility does some extra work that prevents any issues with the addon.

Reply:
Yes, I understand. This was just a temporary change for me to see any stack traces.

Reply:
Okay. I'd add a //FIXME so we don't forget to remove it.